### PR TITLE
Add Sample data and Dashboards for the SQEF

### PR DESCRIPTION
**Pull request by epugh**
### Description
We want to more widely distribute the dashboards (and the nice sample data) for the Search Quality Evaluation Framework (SQEF?) to people.    One way is to add these to the nightly playground environment and maybe soon as one of the standard datasets.   This is the first step by getting these artifacts into a Opensearch repo.

### Issues Resolved
This will help move https://github.com/opensearch-project/opensearch-devops/issues/214 forward.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: search-quality-evaluation-dashboards/README.md
[the remark this answers is not in the document]
Comment (epugh) [reply search-quality-evaluation-dashboards/README.md:1]: i have some text, but getting weird repos errors on push!???

**Comment by epugh** [comment]
What does this mean?
```
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 4 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 1.08 KiB | 1.08 MiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.        
remote: error: GH013: Repository rule violations found for refs/heads/add_sqef_dashboard.        
remote: Review all repository rules at https://github.com/opensearch-project/user-behavior-insights/rules?ref=refs%2Fheads%2Fadd_sqef_dashboard        
remote: 
remote: - Changes must be made through a pull request.        
remote: 
To github.com:opensearch-project/user-behavior-insights.git
 ! [remote rejected] add_sqef_dashboard -> add_sqef_dashboard (push declined due to repository rule violations)
error: failed to push some refs to 'github.com:opensearch-project/user-behavior-insights.git'
```

**Comment by epugh** [comment]
I want to add to the README:

```
The `sample_search_quality_evaluation_data.ndjson` was generated via the `../ubi-data-generator/` script.

```

**Comment by jzonthemtn** [comment]
> What does this mean?
> 
> ```
> Enumerating objects: 7, done.
> Counting objects: 100% (7/7), done.
> Delta compression using up to 4 threads
> Compressing objects: 100% (4/4), done.
> Writing objects: 100% (4/4), 1.08 KiB | 1.08 MiB/s, done.
> Total 4 (delta 3), reused 0 (delta 0), pack-reused 0 (from 0)
> remote: Resolving deltas: 100% (3/3), completed with 3 local objects.        
> remote: error: GH013: Repository rule violations found for refs/heads/add_sqef_dashboard.        
> remote: Review all repository rules at https://github.com/opensearch-project/user-behavior-insights/rules?ref=refs%2Fheads%2Fadd_sqef_dashboard        
> remote: 
> remote: - Changes must be made through a pull request.        
> remote: 
> To github.com:opensearch-project/user-behavior-insights.git
>  ! [remote rejected] add_sqef_dashboard -> add_sqef_dashboard (push declined due to repository rule violations)
> error: failed to push some refs to 'github.com:opensearch-project/user-behavior-insights.git'
> ```

@epugh The problem is your branch was pushed to this repository and is not coming from a fork. Any branch that gets pushed to **this** repository becomes a protected branch and must be committed to via a pull request. So let's merge this branch, and then make the README change in a new pull request from a branch in your fork.

**Pull request merged**